openwisp_ipam/urls.py

```diff
@@ -12,6 +12,7 @@ def get_urls(api_views):
     return [
         path('api/v1/', include((get_api_urls(api_views), 'ipam'), namespace='ipam')),
         path('accounts/', include('openwisp_users.accounts.urls')),
+        path('api/v1/users', include('openwisp_users.api.urls'), namespace='users_api'),
```

Comment:
look at how it's implemented here: https://github.com/openwisp/openwisp-firmware-upgrader/blob/master/tests/openwisp2/urls.py#L19

Reply:
Okay, trying to understand it 😃

Reply:
Hi! I've a doubt! 

is that something like this ? `path('api/v1/users', include((get_api_urls(api_views), 'users_api'), namespace='users_api')),`

Reply:
and are there any other changes?

Reply:
use url instead of path if it doens't work, make sure the namespace is exactly the same as used in the other modules (`users not `users_api`).